bindings/experimental/distrdf/python/DistRDF/Backends/Spark/Backend.py

```diff
@@ -118,16 +130,31 @@ def spark_mapper(current_range):
                 pyspark.SparkFiles.get(ntpath.basename(filepath))
                 for filepath in headers
             ]
-            Utils.declare_headers(headers_on_executor)
+            Utils.distribute_headers(headers_on_executor)
 
             # Get and declare shared libraries on each worker
             shared_libs_on_ex = [
                 pyspark.SparkFiles.get(ntpath.basename(filepath))
                 for filepath in shared_libraries
             ]
-            Utils.declare_shared_libraries(shared_libs_on_ex)
+            Utils.distribute_shared_libraries(shared_libs_on_ex)
+            
+            files_on_executor = [
+                pyspark.SparkFiles.get(ntpath.basename(filepath))
+                for filepath in files
+            ]
+                        
+            pcms_on_ex = [
+                pyspark.SparkFiles.get(ntpath.basename(filepath))
+                for filepath in pcms
+            ]            
```

Comment:
Do we actually need to get these file paths here or can we just remove this code?

Reply:
let me double check

Reply:
ok, so for the files_on_executor - we can remove this, not needed 
but for the pcms the issue happens when we want to load a shared library if it comes from the CompileMacro, then we get a lot of warnings of the type: 
```
Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libEG_rdict.pcm

Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libFitPanel_rdict.pcm

Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libFoam_rdict.pcm

Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libGCocoa_rdict.pcm

Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libGed_rdict.pcm

Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libGenVector_G__GenVector32_rdict.pcm

Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libGenVector_rdict.pcm

Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libGenetic_rdict.pcm

Info in <TCling::LoadPCM>: In-memory ROOT PCM candidate /Users/martaczurylo/Work/distributed_impr/build/lib/libGeomPainter_rdict.pcm
```
and we suppress them by adding the pcms_on_ex. 

I should probably add a test for such a scenario of distributing a shared library too.